src/faebryk/core/core.py

```diff
@@ -664,12 +664,16 @@ def is_mergeable_with(self, other: "Parameter[PV]") -> bool:
             return False
 
     def is_more_specific_than(self, other: "Parameter[PV]") -> bool:
+        from faebryk.library.ANY import ANY
         from faebryk.library.TBD import TBD
 
         if isinstance(self, TBD):
             return False
         if isinstance(other, TBD):
             return False
+        if isinstance(self, ANY):
+            return isinstance(other, ANY)
+
```

Comment:
- if other is any return true
- check that in top
- get most narrow before isinstance